cpp/ql/src/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -161,61 +163,61 @@ class Expr extends StmtParent, @expr {
   predicate mayBeGloballyImpure() { any() }
 
   /**
-   * Holds if this expression is an lvalue. An lvalue is an expression that
+   * Holds if this expression is an *lvalue*. An *lvalue* is an expression that
    * represents a location, rather than a value.
    * See [basic.lval] for more about lvalues.
    */
   predicate isLValueCategory() { expr_types(underlyingElement(this), _, 3) }
 
   /**
-   * Holds if this expression is an xvalue. An xvalue is a location whose
-   * lifetime is about to end (e.g. an rvalue reference returned from a function
+   * Holds if this expression is an *xvalue*. An *xvalue* is a location whose
+   * lifetime is about to end (e.g. an *rvalue* reference returned from a function
    * call).
    * See [basic.lval] for more about xvalues.
    */
   predicate isXValueCategory() { expr_types(underlyingElement(this), _, 2) }
 
   /**
-   * Holds if this expression is a prvalue. A prvalue is an expression that
+   * Holds if this expression is a *prvalue*. A *prvalue* is an expression that
    * represents a value, rather than a location.
    * See [basic.lval] for more about prvalues.
    */
   predicate isPRValueCategory() { expr_types(underlyingElement(this), _, 1) }
 
   /**
-   * Holds if this expression is a glvalue. A glvalue is either an lvalue or an
-   * xvalue.
+   * Holds if this expression is a glvalue. A glvalue is either an *lvalue* or an
+   * *xvalue*.
    */
   predicate isGLValueCategory() { isLValueCategory() or isXValueCategory() }
 
   /**
-   * Holds if this expression is an rvalue. An rvalue is either a prvalue or an
-   * xvalue.
+   * Holds if this expression is an *rvalue*. An *rvalue* is either a *prvalue* or an
+   * *xvalue*.
    */
   predicate isRValueCategory() { isPRValueCategory() or isXValueCategory() }
 
   /**
    * Gets a string representation of the value category of the expression.
    * This is intended only for debugging. The possible values are:
    *
-   * - "lvalue"
-   * - "xvalue"
-   * - "prvalue"
-   * - "prvalue(load)"
+   * - "*lvalue*"
+   * - "*xvalue*"
+   * - "*prvalue*"
+   * - "*prvalue*(load)"
    *
-   * The "prvalue(load)" string is used when the expression is a prvalue, but
-   * `hasLValueToRvalueConversion()` holds.
+   * The "*prvalue*(load)" string is used when the expression is a **prvalue**, but
+   * **hasLValueToRvalueConversion()** holds.
```

Comment:
I preferred backticks here.

Reply:
Ok, if you insist :-)  Like I mentioned above (somewhere), I used **bold** to differentiate QL syntax from C/C++ syntax.